Gemfile.lock

```diff
@@ -1,60 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.0)
+    better_errors (2.5.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (10.0.2)
+    byebug (11.0.1)
     coderay (1.1.2)
     commander (4.4.7)
       highline (~> 2.0.0)
     database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubi (1.7.1)
-    event_sourcery (0.22.0)
+    erubi (1.9.0)
+    event_sourcery (0.23.0)
```

Comment:
### [event_sourcery](https://github.com/envato/event_sourcery)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.22.0 → 0.23.0

[_[change-log](https://github.com/envato/event_sourcery/blob/master/CHANGELOG.md), [source-code](https://github.com/envato/event_sourcery)_]





<details>
<summary>Commits</summary>

A change of **26** commits. See the full changes on [the compare page](https://github.com/envato/event_sourcery/compare/v0.22.0...v0.23.0).

These are the first 10 commits:
- (a7ac051) [Test on Ruby 2.6](https://github.com/envato/event_sourcery/commit/a7ac0512d7791ef804f6abba6db5e7738feb0f6b)
- (9a3c2bf) [Add CI Ruby 2.6 addition to the changelog](https://github.com/envato/event_sourcery/commit/9a3c2bfa745311014218ccaea4b8983983840f38)
- (b6bdf54) [Enable use of Bundle 2.0 in development](https://github.com/envato/event_sourcery/commit/b6bdf54df8e32544c623c7dec02061fa9d6decd0)
- (b1955a5) [Don't test on Ruby 2.2](https://github.com/envato/event_sourcery/commit/b1955a59626e6a1ef57c1a0f8a2deace74e9f76f)
- (80f8211) [Install the latest Rubygems](https://github.com/envato/event_sourcery/commit/80f8211684eea96ad70f546c21604f1b42aaae94)
- (a56bde1) [Add CI Ruby 2.2 removal to the changelog](https://github.com/envato/event_sourcery/commit/a56bde170058122931b75b04d80d705272bce404)
- (e86fe8f) [Merge pull request #212 from envato/orien/bundler-2.0](https://github.com/envato/event_sourcery/commit/e86fe8fc7d0be2b333678ccf8e2e39c27fb84386)
- (aa291fb) [Fix the broken reference to memory store in README](https://github.com/envato/event_sourcery/commit/aa291fb0c1704ff83da6ac112669930bbecbaa27)
- (db02607) [Merge branch 'master' into orien/ruby-2.6](https://github.com/envato/event_sourcery/commit/db02607ea73524ba7289a4903398e47bfb8626b1)
- (4732721) [Default to INFO level](https://github.com/envato/event_sourcery/commit/4732721aa30fde2d025b805212254b68f17ae834)


</details>